CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   `@lumx/core`: fix customization of selected button CSS variables
 -   `@lumx/core`: red/D2, red/N and green/N colors to fix color contrast a11y
 
+### Added
+
+-   DatePicker: add `TextField` for setting year
```

Comment:
No need to mention the textfield. I think it's better to have a more generic, user-oriented description
```suggestion
-   DatePicker: add an input to change the displayed year
```

Reply:
Fixed